src/worldlogic.rs

```diff
@@ -0,0 +1,29 @@
+use crate::world::World;
+use crate::utils::Utils;
+use crate::definitions;
+use std::cell::RefCell;
+use std::rc::Rc;
+
+
+#[derive(Clone)]
+pub struct Worldlogic{
+    world:Rc<RefCell<World>>,
+}
+
+impl Worldlogic {
+    pub fn new(world: Rc<RefCell<World>>) -> Worldlogic {
+        Worldlogic {
+            world
+        }
+    }
+
+    pub fn _sandAction(x:i32,y:i32){
+        const bias = utils::random_sign();
```

Comment:
let not const should define a constant variable in rust
utils must be capitalised

`let bias = Utils::random_sign();`